draft-ietf-quic-transport.md

```diff
@@ -4697,11 +4697,11 @@ active_connection_id_limit (0x000e):
   to store. This value includes the connection ID received during the handshake,
   that received in the preferred_address transport parameter, and those received
   in NEW_CONNECTION_ID frames.
-  Unless a zero-length connection ID is being used, the value of the
-  active_connection_id_limit parameter MUST be no less than 2. If this
-  transport parameter is absent, a default of 2 is assumed.
-  When a zero-length connection ID is being used, the active_connection_id_limit
-  parameter MUST NOT be sent.
+  The value of the active_connection_id_limit parameter MUST be no less
+  than 2. If this transport parameter is absent, a default of 2 is assumed.
+  If an endpoint uses a zero-length connection ID, the
+  active_connection_id_limit value received from its peer is ignored and
+  not used.
```

Comment:
```suggestion
  not used, as NEW_CONNECTION_ID frames can't be sent by a peer that opts for
  a zero-length connection ID.
```

Reply:
I slightly tweaked the suggested change: s/peer/endpoint/ and s/can't/cannot/.